integration/src/main/resources/suites/TempTestSuite.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Temp Test Suite",
```

Comment:
This is temp, should it be committed?

Reply:
Yes, but I could pick a different name. Any suggestions?

It is a kind of scratch suite for running some subset of integration tests in the github runner. I made run-integration-test.yml to do that.

Reply:
sgtm! I missed the new action and wasn't sure if this was for local testing or unmerged branches